lab_1/start.py

```diff
@@ -31,14 +31,14 @@
     top_n = get_top_n_words(dictionary, 3)
     print("Here are top n popular words: ", top_n)
 
-    concordance = get_concordance(string_new, 'sick', 1, 2)
-    print("Concordance for token 'sick' looks like this: ", concordance)
+    concordance = get_concordance(string_new, 'time', 1, 2)
+    print("Concordance for a searched token looks like this: ", concordance)
 
-    adjacent_words = get_adjacent_words(string, 'sick', 1, 2)
+    adjacent_words = get_adjacent_words(string, 'time', 1, 2)
     print("Here are the adjacent words: ", adjacent_words)
 
     #  here goes your logic: calling methods from concordance.py
 
     RESULT = concordance
     # DO NOT REMOVE NEXT LINE - KEEP IT INTENTIONALLY LAST
-    assert RESULT == [['feels', 'sick', 'worn', 'tired']], 'Concordance not working'
+    assert RESULT, 'Concordance not working'

```

Comment:
assert RESULT == [expected result]

if Result = concordance, then put in in [brackets] the expected result
like you had (with 'sick')
`assert RESULT == [['feels', 'sick', 'worn', 'tired']]`